xun/tests/test_graph.py

```diff
@@ -0,0 +1,120 @@
+from xun.functions.graph import CallNode
+from xun.functions.graph import CallNodeSubscript
+
+
+def test_unpack():
+    cn = CallNode('f')
+
+    assert CallNodeSubscript(cn, (0, )) == CallNodeSubscript(cn, (0, ))
+    assert not CallNodeSubscript(cn, (0, )) == CallNodeSubscript(cn, (1, ))
```

Comment:
Do we need __eq__ for CallNodeSubscript?

Reply:
Yes, I think we do, in order to compare to the expected values